client/state/data-layer/wpcom/read/following/mine/index.js

```diff
@@ -19,6 +19,19 @@ import { errorNotice } from 'state/notices/actions';
 const ITEMS_PER_PAGE = 200;
 const MAX_ITEMS = 2000;
 
+export function toValidId( val ) {
```

Comment:
I think if stay smart about naming the functions, we shouldn't hesitate to throw lots of util functions into `reader/util`.  If we ever run into an issue where we feel that the file is too large, then we can think of how to split it up better.  potentially even having a util folder. 

In general creating more reusable functions for common operations will be a net win for us (like what we started doing with postKeys)